spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe ForgotPasswordsController do
+  describe "POST create" do
+    context "with blank input" do
+      it "redirects to the forgot password page" do
+        post :create, email: ''
+        expect(response).to redirect_to forgot_password_path
+      end 
+      it "shows an error message" do
+        post :create, email: ''
+        expect(flash.now[:error]).to eq("Email cannot be blank.")
```

Comment:
Since you are using `flash` in the controller and not `flash.now` I would mirror what you are actually using if only for clarity's sake. The test actually passes either way regardless of which way you do this.